packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,258 @@
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
+
+```jsx
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom'; // MemoryRouter or HashRouter could also be used
+import Base from 'terra-base';
+import ApplicationLayout from 'terra-application-layout';
+
+const MyApp = () => (
+  <BrowserRouter>
+    <Base locale="en-US">
+      <ApplicationLayout
+        nameConfig={nameConfig}
+        utilityConfig={utilityConfig}
+        routingConfig={routingConfig}
+        navigationItems={navigationItems}
+        extensions={<ApplicationExtensions />}
+        indexPath={indexPath}
+      />
+    </Base>
+  </BrowserRouter>
+);
+```
+
+## Props
+
+### `app`
+#### Is Required: `false`
+
+By default, the ApplicationLayout will render itself within a `ModalManager` and provide modal presentation capabilities to all of its child components. However, the ApplicationLayout accepts an AppDelegate prop (as `app`) that it will fall back to, if necessary. If no `app` prop is provided, the ModalManager provided by the ApplicationLayout will handle all disclosure requests by the ApplicationLayout's children.
+
+### `extensions`
+#### Is Required: `false`
+
+The `extensions` prop allows consumers to render an element within the ApplicationLayout's extensions region. The `extensions` content will be rendered in various ways depending on the current breakpoint. The `extensions` element will receive `app` and `layoutConfig` props from the ApplicationLayout, allowing it to interact with and respond to changes within the ApplicationLayout.
+
+```jsx
+import { Utils } from 'terra-application-layout';
```

Comment:
This may read better as `import ApplicationLayout, { Utils } from 'terra-application-layout';` as users likely need to import both when they use Utils.

Reply:
I see what you're saying, but at the same time I don't want code in these examples to immediately fail linting if copied. This example is straight out of my extensions example file, which does not use or import the ApplicationLayout.